release.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/sh
 
 version="$1"
+versioned_release_branch="release-${version}"
```

Comment:
What do you think about making branch = tag?

Reply:
I'd like to separate them into different name. `git push` can be used for both branch and tag. If both branch and tag has the same name, your command would not work. Also, I think naming them differently helps us to understand one is branch, another is tag.

an example why `git push` would not work if both branch and tag exist:
```
// in a directory with git
$ git checkout -b ycchou-test-version
A       src/build-opencensus-cpp
Switched to a new branch 'ycchou-test-version'
$ git tag ycchou-test-version
$ git push origin ycchou-test-version
error: src refspec ycchou-test-version matches more than one
error: failed to push some refs to 'git@github.com:StevenYCChou/metadata-agent.git'
```